Gemfile

```diff
@@ -106,3 +106,7 @@ gem 'jbuilder'
 
 # Pin mustermann to Ruby 2.1 compatible
 gem 'mustermann' , '~> 0.4'
+
+#Performance testing
+gem 'rails-perftest'
```

Comment:
Could we move this inside the "test" section of the gemfile? Thank you!

Reply:
Yes, will move it!

Reply:
To clarify, by 'test' section, are you referring to inserting it below 'group :test, :development do', but outside of that that block in the gem file?

Reply:
It can go inside that `group :test, :development do` block, thanks!